operations/preinstall/Preinstall.md

```diff
@@ -0,0 +1,22 @@
+## Pre-Install Steps
+
+This section provides information on what needs to be done before
+an initial install of CSM.
+
+### River BMC Credentials
+
+It is necessary to set the default credentials of all River BMCs so that
+CSM Hardware Management can interact with Redfish.
+
+This procedure is outlined in [Change River BMC Credentials](Change_River_BMC_Credentials.md).
+
+### ServerTech PDU Default Credentials
+
+ServerTech PDUs must have their default credentials set as well.  These are
+not native Redfish devices; they have a proprietary interface which is
+abstracted by the HMS Redfish Translation Service.
+
+To set ServerTech PDU default credentials, follow the procedure outlined in
+[Change ServerTech PDU Credentials](Change_ServerTech_PDU_Credentials.md).
+
+
```

Comment:
I think we should also link in the  `Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials` procedure so we can get coverage of the Liquid cooled Chassis and Node BMCs. https://github.com/Cray-HPE/docs-csm/blob/main/operations/security_and_authentication/Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md

Though CASMHMS-5164 is only about River/Air cooled hardware, I think referencing that procedure for mountain hardware here would be good.

Reply:
Done.